conans/client/conf/__init__.py

```diff
@@ -25,6 +25,12 @@
     apple-clang:
         version: ["5.0", "5.1", "6.0", "6.1", "7.0", "7.3"]
         libcxx: [libstdc++, libc++]
+    arm-none-eabi-gcc:
+        version: ["5.2"]
+        libcxx: [None]
```

Comment:
If there is only one setting and it is None, we can omit this setting at the moment, and add it later if necessary. The value "None" in the future is allowed to not change the hash of existing package binaries, so it shouldnt be a problem.